cg/constants/compression.py

```diff
@@ -112,21 +112,36 @@
 
 # List of cases used for validation that we should skip
 RNAFUSION_VALIDATION_CASES = [
-    "cuddlyhen",  # downsampled seracare commercial sample
-    "oncrab",
+    "ableheron",
+    "acecoyote",
+    "ampleray",
+    "bossmink",
+    "cuddlyhen",
     "daringowl",
+    "expertboar",
+    "finerracer",
     "growndoe",
-    "stablemoray",
-    "truemole",
-    "rightmoray",
+    "guidedfeline",
     "handyturtle",
-    "inlab",
-    "honestswine",
+    "hardygrouse",
     "holyrodent",
-    "expertboar",
-    "bossmink",
-    "ampleray",
+    "honestswine",
+    "inlab",
+    "justburro",
+    "movedmule",
     "nearbyjoey",
+    "oncrab",
+    "politeglider",
+    "rareosprey",
+    "rightmoray",
+    "stablemoray",
+    "starnewt",
+    "tendergoose",
+    "truemole",
+    "truepigeon",
+    "valuedfowl",
+    "vocallocust",
+    "wantedsawfly",
```

Comment:
Spontaneously, it seems like a bad idea to hard code a list with these generated id:s. Is it not possible to add a check on the case wherever the compression is done to see if it is a RNA fusion case instead?

Reply:
I agree. I strongly dislike this way. 
To me, this information should be in statusDB with a column that is validation=bool or something similar. 
It's also extremely prone to be forgotten to update, like it keeps happening. 
If you have time and want to refactor this I'd be happy

Reply:
Yes, I could refactor this. Seems like there is a bunch of other validation cases for other pipelines. Not sure if/how to prioritise it. Added an issue for it so it will hopefully be picked up at some point https://github.com/Clinical-Genomics/cg/issues/2607